install/playbooks/roles/borg-backup/templates/backup-key

```diff
@@ -1 +1 @@
-{{ backup_key }}
+{{ backup_key }}
```

Comment:
Is this voluntary? Since the `\n\r` is removed by the code now.
Anyway, it is harmless.

Reply:
Not really.
One of the change in the Python code is to read the repository backup key, but remove the carriage returns and new line characters.
An issue happened when I redeployed the backup code, the key was different (i.e. with an extra carriage return) and the class complained and could not read the archive any more!